# PID Control Parameters

kp_lin: 0.5
kd_lin: 0.01
ki_lin: 0.0

kp_ang: 0.5
kd_ang: 0.01
ki_ang: 0.0

test_goal: [-3,-3]

MAX_VEL: 0.3

SAMPLING_RADIUS: 0.5
GOAL_EPS: 0.5

